buildtest/cli/compilers.py

```diff
@@ -47,31 +47,44 @@ def compiler_test(configuration):
     Args:
         configuration (buildtest.config.SiteConfiguration): An instance of SiteConfiguration class
     """
+    pass_compilers = []
+    fail_compilers = []
+
     bc = BuildtestCompilers(configuration=configuration)
     bc.find_compilers()
 
-    table = Table(title="Compilers Test Pass")
-    table.add_column("No.", style="cyan", no_wrap=True)
-    table.add_column("Compiler Name", style="green")
-    table.add_column("Status", justify="right")
+    for name in bc.compilers:
+        for module in bc.compilers[name]:
+            if bc.compilers[name][module].get("module", None):
+                cmd = Module(module, debug=False)
```

Comment:
i think this line is not correct you should be doing something like `bc.compilers[name][module]['module'].get('load'
) to get a list of modules that get passed to the `Module` class.

Reply:
That's right if we have the condition `if not bc.compilers[name][module].get("module"):` then we should consider compiler as pass and append to pass list otherwise we test the compiler modules

Reply:
Done!